amespahdbpythonsuite/tests/test_spectrum.py

```diff
@@ -128,3 +128,16 @@ def test_getset(self, test_spectrum):
     def test_write_spectrum(self, test_spectrum, test_path):
         test_spectrum.write(f"{test_path}.tbl")
         assert exists(f"{test_path}.tbl")
+
+    def test_mcfit(self, test_transitions):
+        file = resource_filename("amespahdbpythonsuite", "resources/galaxy_spec.ipac")
+        obs = observation.Observation(file)
+        spectrum = test_transitions.convolve(
+            grid=1e4 / obs.spectrum.spectral_axis.value,
+            fwhm=15.0,
+            gaussian=True,
+            multiprocessing=False,
+        )
+        mcfit = spectrum.mcfit(obs, samples=10)
+        assert mcfit.mcfits[0].method == 'NNLC'
```

Comment:
Possibly a assert ```isinstance(mcfit, MCFitted)``` suffices, as we're not testing the inner workings of MCFitted here.